webpack.config.client.js

```diff
@@ -69,6 +73,16 @@ module.exports = ({ resolvePath, IS_CI, IS_PROD, START_DEV_SERVER }) => {
       new webpack.DefinePlugin({
         'process.env': getClientEnvVars(getEnv()),
       }),
+      new webpack.NormalModuleReplacementPlugin(
```

Comment:
This is a neat solution, but we should probably place a comment explaining it very explicitly, as we've documented our webpack pretty thoroughly so far.